tee-worker/scripts/litentry/release/prepare.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -euo pipefail
+
+# Edit the following variable(s):
+WORKER_DIR=/tmp/worker/
```

Comment:
This is not a choice due to `/tmp/...`, I assume it will be deleted once the server is restarted? No matter on purpose or not

Reply:
change to fixed path: /opt/worker